encord/objects/ontology_labels_impl.py

```diff
@@ -857,6 +858,26 @@ def __lt__(self, other) -> bool:
         return self.classification_hash < other.classification_hash
 
 
+@dataclass
+class BundledGetRowsPayload:
+    uids: List[str]
+    get_signed_url: bool
+    include_object_feature_hashes: Optional[Set[str]]
+    include_classification_feature_hashes: Optional[Set[str]]
+    include_reviews: bool
+
+
+@dataclass
+class BundledCreateRowsPayload:
+    uids: List[str]
+
+
+@dataclass
+class BatchSaveRowsPayload:
```

Comment:
Given that the lists have to have an equal length, you may want to consider having a `list[BatchSaveRowsPayload]` where each dataclass has a single uid and payload.